spec/Section 3 -- Type System.md

```diff
@@ -371,6 +361,43 @@ that type) then it must not be included.
 When representing a GraphQL schema using the type system definition language,
 all built-in scalars must be omitted for brevity.
 
+**Custom Scalars**
+
+GraphQL services may use custom scalar types in addition to the built-in
+scalars. For example, a GraphQL service could define a scalar called `Time`
+which, while serialized as a string, conforms to ISO-8601. When querying a field
+of type `Time`, you can then rely on the ability to parse the result with an
+ISO-8601 parser and use a client-specific primitive for time. Another example of
+a potentially useful custom scalar is `URL`, which serializes as a string, but
+is guaranteed by the server to be a valid URL.
+
+When defining a custom scalar, GraphQL services should provide a specification
+URL via `specifiedBy` introspection field or `@specifiedBy` directive. This URL 
+must link to a human-readable specification of the data format, serialization, 
+and coercion rules for the scalar. For example, a GraphQL service providing a 
+`UUID` scalar may link to RFC 4122, or some document defining a reasonable 
+subset of that RFC. If a scalar specification URL is present, services, clients 
+and tools that are aware of it should conform to its described behavior.
+
+```graphql example
+scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
+scalar URL @specifiedBy(url: "https://tools.ietf.org/html/rfc3986")
+```
+
+Custom scalar specifications should provide a single, stable format to avoid 
+ambiguity. If the linked specification is in flux, the service should link to a 
+fixed version rather than to a resource which might change.
+
+Custom scalar specification URLs should not be changed once defined; as doing so 
```

Comment:
semi-colon seems incorrect here, I think a comma is fine?

Reply:
Or, drop the word "as" to make the second clause a complete sentence.

Reply:
Great catch, thank you!